config/routes.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  root 'message#index'
+
+  get 'greeting/random', to: 'message#greeting' 
+  # Defines the root path route ("/")
+  # root "articles#index"
+end
```

Comment:
- [ ] Great job so far 👏🏻 , however. You don't have to delete the main static route Kindly bring it back and make sure:
1. the created route is marked as `root`. :x: 
2. A controller with one action was created :x: 
3. A view for the root route is created (`root#index`) :x: 
   * It should include only `<div id="root">` The rest should be done with React and Redux. 😉